### PR TITLE
fix: UI bugs in provisioning

### LINKED ISSUE #6789: Publish modal's comment text box needs some ux love
## Describe the bug
When you publish a bot using an existing publishing profile, you'll be prompted with a "Publish" modal where the profile name, publishing target and publishing comment can be added before invoking the actual publish action.

The comments text field is oddly small.  See screen shot.

## Version
Version:  1.4.0-nightly.232669.4daad5c
Electron: 8.2.4
Chrome: 80.0.3987.165
NodeJS: 12.13.0
V8: 8.0.426.27-electron.0

## Browser
- [x] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS
- [x] macOS
- [ ] Windows
- [ ] Ubuntu

## To Reproduce

1. Open an existing bot project.  
2. Create a publishing profile, if the project doesn't yet have one.  This will perform the necessary provision and deployment steps.
3. With a fully functioning publishing profile, nav to the Publish your bots page.
4. Select the bot to publish, ensure you have your publishing target selected (should already be selected)
5. Click the "Publish selected bots" top-level menu item to invoke the publishing flow.  This will display the Publish modal.
6. Notice the comments edit box.  

## Expected behavior
In general, we seem to have overly narrow dialogs in Composer.  In this particular modal, the edit box doesn't even show the full help text.  

## Screenshots
Screen_Shot_2021-04-08_at_11_39_54_AM

### LINKED ISSUE #6886: Configure resource modal fit and finish
Frame 61659

Original design > https://www.figma.com/file/J3oIdXDmeBQbAe4a4PYbQc/Jaskeerat-s-files?node-id=5319%3A86501

### LINKED ISSUE #6887: Fit and Finish - Broken link  in Create Publishing Profile for Azure
See video below>

The Microsoft azure link & the learn more link the in the provisioning modal is broken.

https://user-images.githubusercontent.com/67944302/114478987-5bb5ed00-9bb4-11eb-9947-70191475b465.mp4

### LINKED ISSUE #6888: provisioning modal fit and finish
Can we only make the read me section scrollable and fix the section with radio buttons? Creation modal follows that interaction pattern..

https://user-images.githubusercontent.com/67944302/114479280-16de8600-9bb5-11eb-9d17-1dbedb0ecbe0.mp4

### LINKED ISSUE #6889: import existing resources fit and finish
Needs copy updates ( see below)
Needs back button

Frame 61660

Original design - https://www.figma.com/file/J3oIdXDmeBQbAe4a4PYbQc/Jaskeerat-s-files?node-id=5502%3A822

### LINKED ISSUE #7126: Inconsonant grammer on "Create new LUIS resources" screen
## Describe the bug

The "resource group" text has a ":" on it. 
LUIS region does not. 
Resource name does not. 

They should all be the same. 

## Version

Version:  1.4.0-nightly.236875.b9e46c2
Electron: 8.2.4
Chrome: 80.0.3987.165
NodeJS: 12.13.0
V8: 8.0.426.27-electron.0

## To Reproduce
Look at the text on the "Create new LUIS resources" screen. 

Observe the needed copy-editing. 

image

### LINKED ISSUE #7132: QnA Resource vs Resources
## Describe the bug

Of the 6 mentions of QnA Resources on this dialog, they break down as:

1. 2 Plural
2. 4 Singular

Consider standardizing. 

image

## Version

Version:  1.4.0-nightly.236875.b9e46c2
Electron: 8.2.4
Chrome: 80.0.3987.165
NodeJS: 12.13.0
V8: 8.0.426.27-electron.0

### LINKED ISSUE #7133: Resource (LUIS / QnA) Dialogs can be closed during resource create
## Describe the bug

While creating resources (QnA / LUIS) the Back/Next/Cancel buttons are greyed out. 

However the "X" in the top right remains functional. Closing this during resource creation seems to have unexpected consequences. 

image

## Version

Version:  1.4.0-nightly.236875.b9e46c2
Electron: 8.2.4
Chrome: 80.0.3987.165
NodeJS: 12.13.0
V8: 8.0.426.27-electron.0

## Browser

- [ ] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [ ] Windows
- [ ] Ubuntu

### PR DESCRIPTION
**Pull request by pavolum**
## Description

Various fixes for UI bugs in provisioning

## Task Item

fixes #6887 : alias link added and Link on click fixed with blank target
fixes #6888 : Fixed scrolling in provisioning modal to only scroll on detail view as opposed to whole modal
fixes #6789 : increased size of publish confirm modal to support larger comment text field
fixes #6886 : Updating copy and back button in config resource modal
fixes #6889: Updating copy and adding button in dialog footer for `import existing resource` modal
fixes #7132: string standardization
fixes #7133: disable modal exit while luis and qna are in mid provisioning
fixes #7126: remove semi colen in luis modal

### CODE REVIEW

**Review by a-b-r-o-w-n — COMMENTED** [review]
Can you also update the locale file by running `yarn l10n`?

File: Composer/packages/client/src/components/ManageLuis/ManageLuis.tsx [hunk 1]
```diff
@@ -598,7 +598,7 @@ export const ManageLuis = (props: ManageLuisProps) => {
         modalProps={{
           isBlocking: true,
         }}
-        onDismiss={props.onDismiss}
+        onDismiss={!loadingLUIS ? props.onDismiss : () => {}}
```
Comment (beyackle) [inline Composer/packages/client/src/components/ManageLuis/ManageLuis.tsx:601]:
```suggestion
        onDismiss={loadingLUIS ? () => {} : props.onDismiss}
```

File: Composer/packages/client/src/components/ManageQNA/ManageQNA.tsx [hunk 2]
```diff
@@ -518,14 +518,14 @@ export const ManageQNA = (props: ManageQNAProps) => {
...
-          title: currentPage === 2 ? formatMessage('Create new QnA resources') : formatMessage('Select QnA keys'),
+          title: currentPage === 2 ? formatMessage('Create new QnA resource') : formatMessage('Select QnA keys'),
         }}
         hidden={props.hidden}
         minWidth={480}
         modalProps={{
           isBlocking: true,
         }}
-        onDismiss={props.onDismiss}
+        onDismiss={!loadingQNA ? props.onDismiss : () => {}}
```
Comment (beyackle) [inline Composer/packages/client/src/components/ManageQNA/ManageQNA.tsx:528]: Same here; if you're writing a `?:` thing with a "not" in the condition, just remove the "not" and swap the two results.

File: extensions/azurePublish/src/components/azureProvisionDialog.tsx [hunk 3]
```diff
@@ -96,12 +96,12 @@ const PageTypes = {
 
 const DialogTitle = {
   CHOOSE_ACTION: {
-    title: formatMessage('Configure resources'),
+    title: formatMessage('Configure resources to your publishing profile'),
```
Comment (beyackle) [inline extensions/azurePublish/src/components/azureProvisionDialog.tsx:99]: While we're making grammatical changes, this phrasing seems weird. Maybe "Configure resources in your publishing profile"? Or "within"? "Using"? I'm not 100% clear on what the relationship is, but "configure to" doesn't sound right in any case.

**Review by beyackle — CHANGES_REQUESTED** [review]

File: Composer/packages/client/src/components/ManageLuis/ManageLuis.tsx (hunk 1 above)
Comment (pavolum) [reply Composer/packages/client/src/components/ManageLuis/ManageLuis.tsx:601]: Handled

File: extensions/azurePublish/src/components/azureProvisionDialog.tsx (hunk 3 above)
Comment (pavolum) [reply extensions/azurePublish/src/components/azureProvisionDialog.tsx:99]: I will run this by design and change accordingly

Comment (pavolum) [reply extensions/azurePublish/src/components/azureProvisionDialog.tsx:99]: Design has a task all up to review copy and will resolve changes in a seperate PR, I have provided them your input for this one. For this PR we will leave it as is in anticipation of a full PR around copy

**Review by cwhitten — APPROVED** [review]

**Pull request merged**